src/main/java/uz/bookstore/bookstore/dto/ResultMessage.java

```diff
@@ -0,0 +1,14 @@
+package uz.bookstore.bookstore.dto;
```

Comment:
we do not need this class. Because our response will change according to entity class.